src/core/addonManager.js

```diff
@@ -92,7 +92,9 @@ export function getAddon(
       if (!addon) {
         throw new Error('Addon not found');
       }
-      log.info('Add-on found', addon);
+      // eslint-disable-next-line amo/only-log-strings
+      log.debug('Add-on found', addon);
```

Comment:
The more I think about the ESLint rule, the more I believe `debug` should not be taken into account. WDYT?